lang/extract_json_strings.py

```diff
@@ -205,6 +205,7 @@ def extract_harvest(item):
 
 def extract_bodypart(item):
     outfile = get_outfile("bodypart")
+    writestr(outfile, item["name"])
```

Comment:
Unless I'm missing something this is unrelated to the rest of the PR.
it's also redundant with the next line.

Reply:
I thought so too, so I deleted this line in #32655  , but not long after I found that the translation of the "body part" in the "effects" disappeared, and the translation of the "body part" in the "clothes" was still there.
After querying I found this code.

```
std::string body_part_name( body_part bp, int number )
{
    const auto &bdy = get_bp( bp );
    return ngettext( bdy.name.c_str(),
                     bdy.name_multiple.c_str(), number );
}

std::string body_part_name_accusative( body_part bp, int number )
{
    const auto &bdy = get_bp( bp );
    return npgettext( "bodypart_accusative",
                      bdy.name.c_str(),
                      bdy.name_multiple.c_str(), number );
}
```

The author creates two similar functions, both of which are used for the return bodypart, providing the same msgid, but providing context for one of them, context = "bodypart_accusative".


I guess the "body part" of some languages ​​is not the same as using accusative and nominative.

If we need to simplify, deleting the second sentence should not cause problems, at least in Chinese and English.

```
    writestr(outfile, item["name"])
    writestr(outfile, item["name"], context="bodypart_accusative")
```

Reply:
I see, they're the same in English but might not be in other languages, thanks for the correction.